ocean_lib/models/v4/erc20_token.py

```diff
@@ -147,15 +147,19 @@ def finish_multiple_order(
             from_wallet,
         )
 
+    @enforce_types
```

Comment:
No need to put `@enforce_types` annotation to these functions if it's already set at the above the class declaration. It applies `enforce_types` to all the functions inside the class.